### ISSUE TITLE
contacts profile name showing differently on android.

### ISSUE THREAD

**Issue by madLyfe** [wireapp/wire-android#1351]
## The problem
when a contact changes their profile name, it is not synced correctly on my phone. their profile picture is, though. everything is correct on the web and desktop app.
## Environment
* Wire version that exhibits the issue: 3.4.382
* Last Wire version that did not exhibit the issue (if applicable): N\A
* Android OS version used to run Wire: 8.1 dec security patch
* Is this a custom firmware or a stock one: stock
* Mobile phone model/manufacturer: P2XL
* Mobile network type (EDGE/LTE/Wi-Fi/Offline): all of the above
## Details
connect with contact. contact changes profile name.

**Comment by someoneEsle** [comment]
What happens if you clear the cache and restart the app?

**Comment by madLyfe** [comment]
nope that doesn't fix it. also, I guess my friend is showing my old profile name that I changed along time ago. and changed it a few weeks ago. he just joined today.

**Comment by someoneEsle** [comment]
> I guess my friend is showing my old profile name that I changed along time ago

Let me understand, your friend joined today/yesterday, but he sees your old name? A name you changed a few weeks ago?

**Comment by madLyfe** [comment]
yep. I also see my old name in my archive list and in my contacts list and can't get rid of it on my phone. the contact isn't there on the web/desktop in archive but is still in my contacts list on the web/desktop.

**Comment by alien2003** [comment]
the same happens to me

**Comment by madLyfe** [comment]
when i go to my connections(web) or contacts(android) it still shows my original name i signed up with and not my current name. are others still getting this?

**Comment by someoneEsle** [comment]
I'm not sure what you mean when you say that you see your old name in your "connections(web) or contacts(android)". How can you see yourself in your contacts list? Or are you using two accounts?

What happens if you try to change your name again? Do you see it updated across your devices?

**Comment by madLyfe** [comment]
> I'm not sure what you mean when you say that you see your old name in your "connections(web) or contacts(android)". How can you see yourself in your contacts list? Or are you using two accounts?

i dont know how i can see myself in the contacts list. but the original name i signed up with is there. im not using two accounts, i changed my profile/user name after i made the account though.

> What happens if you try to change your name again? Do you see it updated across your devices?

i just changed both my profile/user name and it shows up normally on web and android for me. i still see my original name i signed up with under my contacts list and a group chat(that im currently active in) still shows my original profile/user name in its member list because it was made before i first changed my profile/user name.

maybe i can start a secure chat with you and send you some screenshots?

**Comment by someoneEsle** [comment]
Please allow me to ask some more questions: if you log into a new device (another android phone, or another web client) do you still see your old name in the contacts list? What happens when you tap it? Does it open a conversation that looks like an empty group (like, a group where you're the only participant), or does it look like any other conversation with another person? Does this account have the same @username as your current account?
Thanks for your help

**Comment by madLyfe** [comment]
yes i see my old name in a new web client(different browser than i normally use). one thing that is consistent is i dont see the @username for my user name in the contacts list.

when clicked it looks like a conversation i would have with another person. just without an @username.

like i was saying, maybe this would be easier to wrap your head around if i can send you some screenshots?

**Comment by someoneEsle** [comment]
You can file the bug [here](https://support.wire.com/hc/en-us/requests/new?ticket_form_id=101615) and attach your screenshots. When you send your report, please link to this github issue so that Support will contact me. Thanks!
